scripts/forger/seed/download_seed.sh

```diff
@@ -24,23 +24,24 @@ extended_content_length=$((content_length + content_length / 10))
 total_size=$((2 * extended_content_length))
 
 {
+  echo "Starting seed process..."
   echo "Seed file size: ${content_length} bytes"
   echo "Minimal required storage size: ${extended_content_length} bytes"
   echo "Recommended storage size: ${total_size} bytes"
-} >>"${seed_dir}/seed.log"
+} | tee -a "${seed_dir}/seed.log"
```

Comment:
I think you are missing stderr to stdout redirection everywhere before the pipe `2>&1`